src/main/webapp/script.js

```diff
@@ -48,16 +48,88 @@ class Measure {
         buttonDrop.onclick = this.showDropMenu;
         this.button = buttonDrop;
 
-        //Create the links
-        var Piano_A = document.createElement("a");
-        Piano_A.innerText = "Piano A";
-        Piano_A.href="#";
-        Piano_A.onclick = this.pianoA;
 
-        var Piano_B = document.createElement("a");
-        Piano_B.innerText = "Piano B";
-        Piano_B.href="#";
-        Piano_B.onclick = this.pianoB;
+        // Sound Options Drop Down Creation: [from free mp3 downloaders online]
+            
+        // Option 1: Piano Dramatic   
+        var sound_option_1 = document.createElement("b");
+        sound_option_1.innerText = "option 1";
+        sound_option_1.href="#";
+
+        // creating audio controls to bind to this sound option 
+        var sound_1      = document.createElement('audio');
+        sound_1.id       = 'piano_drama_sound';
+        sound_1.controls = 'controls';
+        sound_1.src      = 'sounds/piano_dramatic.mp3';
+        sound_1.type     = 'audio/mpeg';
+        sound_option_1.appendChild(sound_1);
+
+        // creating onlick connection to send through the corresponding song 
+        sound_option_1.onclick = this.pianoA(sound_1);
```

Comment:
Very neat! Given that you write the same code for all 5 options, I would suggest making a function to avoid redundancy.